src/Common/src/System/IO/PathInternal.Unix.cs

```diff
@@ -8,6 +9,9 @@ namespace System.IO
     /// <summary>Contains internal path helpers that are shared between many projects.</summary>
     internal static partial class PathInternal
     {
+        internal static readonly char[] InvalidPathChars = { '\0' };
+        internal static readonly char[] InvalidPathCharsWithAdditionalChecks = InvalidPathChars; // no additional checks on Unix
```

Comment:
Er, nothing for the additional checks. :) Path.GetInvalidPathChars() uses the first. Missed that I didn't move the usage.